styles.css

```diff
@@ -16,22 +16,22 @@
             "footer";
 
 }
-    /*alles wat hierin staat is voor GROTer dan mobiel , tablets en desktops. ALLEEN de properties die veranderen*/
-    @media screen and (min-width: 576px) {
-        .grid-container {
-            grid-template-columns: 1fr 1fr 1fr;
-            grid-template-rows: 50px 240px 100px auto 50px;
-            grid-template-areas:
+/*alles wat hierin staat is voor GROTer dan mobiel , tablets en desktops. ALLEEN de properties die veranderen*/
+@media screen and (min-width: 576px) {
+    .grid-container {
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 50px 240px 100px auto 50px;
+        grid-template-areas:
         "header large-sidebar large-sidebar"
         "main large-sidebar large-sidebar"
         "main mid-bar small-side-bar"
         "main mid-bar ."
         "footer footer footer";
 
-        }
-
     }
 
+}
+
 .grid-item {
```

Comment:
Voor de rest lijk die goed, super efficient dat je een grid item class had gemaakt om de kleur, flex positions enzo te zetten en dan id's per block had gemaakt voor de grid areas om minder duplicate code te hebben! Had ik zelf niet aan gedacht, dus gedankt voor de tip.

Je kon ook de background-color in de grid item class doen omdat alle blocks de zelfde kleur hebben. (Als je dan een block toch en andere kleur wil geven, kan je het dan voor die id het nogsteeds een background color geven en dan zou die nog steeds veranderen denk ik omdat id meer specifiek is dan class)

Reply:
top, dank voor de feedback